service/src/main/resources/db/migration/V119__daycare_group_start_before_end.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE daycare_group ADD CONSTRAINT start_before_end CHECK ((start_date <= end_date));
```

Comment:
Do we want to allow `start_date == end_date`?

Reply:
yes, technically it is ok (end date is inclusive)

Reply:
It's probably not a very meaningful case, but semantically it's a valid date range so there's no reason to forbid it